app/static/js/base.js

```diff
@@ -33,37 +32,74 @@ function msgToast(head, body){
   $("#toast-header").html(head)
   $("#toast-body").html(body)
   toastList[0].show()
+  $("#liveToast").delay(2000).fadeOut();
```

Comment:
Could we make the fadeout longer? At this length is looks like something people could miss if they're a bit distracted.